downstream/modules/hub/proc-configure-automation-hub-server-gui.adoc

```diff
@@ -8,40 +8,40 @@ Configure Red Hat automation hub as your primary source of content by using the
+Creating a new token revokes any previous tokens generated for {PrivateHubName}. Ensure that you update any {ControllerName} or scripts that you created with the previous token.
 ====
 
 .Procedure
 
-. Navigate to your Automation Controller.
+. Navigate to your {ControllerName}.
 . Create a new credential.
-.. Click btn:[Add] from the **Credentials** screen.
-.. Enter the name for your new credential in the **Name** field.
+.. Navigate to menu:Resources[Credentials].
+.. Click btn:[Add].
+.. Enter the name for your new credential in the *Name* field.
 .. Optional: Enter a description and enter or select the name of the organization with which the credential is associated.
-.. Under **Organization**, select the organization that you would like to use the credential for.
-.. Select **Ansible Galaxy/Automation Hub API Token** as the credential type.
-.. Under **Type Details**, enter the **Galaxy Server URL**, **Authentication Server URL**, and **API Token** created in the Prerequisite.
+.. Under *Organization*, select the organization that you would like to use the credential for.
+.. Select *Ansible Galaxy/Automation Hub API Token* as the credential type.
+.. Under *Type Details*, enter the *Galaxy Server URL*, *Authentication Server URL*, and *API Token* previously created.
 .. Click btn:[Save].
-. Select the credential you created from the Organization’s **Galaxy Credentials** field.
+. Select the credential you created from the Organization’s *Galaxy Credentials* field.
 .. Navigate to menu:Access[Organizations].
 .. Select the organization where you’d like to add your Galaxy credentials.
 .. Click btn:[Edit].
-.. Under Galaxy Credentials, click the **Search** icon.
+.. Under Galaxy Credentials, click the btn:[Search] icon.
 .. Select the credential you created for automation hub, and place it at the beginning of the list.
 .. Optional: If you have a secondary source of content, such as Ansible Galaxy, place this credential after the credential you created for automation hub.
 .. Click btn:[Select].
 .. Click btn:[Save].
 
 .Verification
 
-To validate the credential, update an existing source control management (SCM)-based project by selecting the project and clicking the **Refresh** icon.
+To validate the credential, update an existing source control management (SCM)-based project by selecting the project and clicking the btn:[Sync Project] icon.
 
 . Navigate to your project repository.
 . Select a project that uses a `collections/requirements.yml` file.
-. Update the project by clicking the refresh icon [refresh icon].
+. Update the project by clicking the btn:[Sync Project] icon.
 
-If the status of the project is **Successful**, then the credential is valid.
+If the *Status* of the project is *Successful*, then the credential is valid.
 
 You have now configured automation hub as your primary server using the web console, and you can proceed to download and install supported collections.
```

Comment:
We should use the attribute here--{HubName}

Reply:
Fixed.